projects/js-packages/api/index.jsx

```diff
@@ -83,7 +83,7 @@ function JetpackRestApiClient( root, nonce ) {
 			cacheBusterCallback = callback;
 		},
 
-		registerSite: ( deprecated, redirectUri, from ) => {
+		registerSite: ( redirectUri, from ) => {
```

Comment:
For future reference, we can't remove `deprecated` argument just yet for backward compatibility reasons.
Plugins don't always use the latest version of JS packages if multiple versions available from different plugins.